### PR TITLE
US13362 | Christian | css, html update DDK to match current version

### PR DESCRIPTION
**Pull request by christiancrawford**

### CODE REVIEW

File: src/app/components/pin-details/gathering/gathering.html [hunk 1]
```diff
@@ -31,11 +31,11 @@
...
       </div>
 
-      <div class="btn-group push-top">
-        <input *ngIf="app.isSmallGroupApp() && isLeader" type="button" class="btn btn-secondary" value="Edit Group" (click)="onEditGroupClicked(pin.gathering.groupId)">
-        <div class="font-size-smaller push-top">
-          <a class="pointer" *ngIf="isLeader && !isInConnectApp" (click)="onEndGroupClicked()">End group</a>
-        </div>
+      <div class="push-top">
+        <button class="btn btn-white btn-outline" *ngIf="app.isSmallGroupApp() && isLeader" type="button" value="Edit Group" (click)="onEditGroupClicked(pin.gathering.groupId)">Edit Group</button>
+        <button class="btn btn-cyan-light btn-outline pointer" *ngIf="isLeader && !isInConnectApp" (click)="onEndGroupClicked()">
```
Comment (zarahzachz) [inline src/app/components/pin-details/gathering/gathering.html]: `btn-cyan-light` isn't supported by the DDK. You may want to discuss the design direction here with @amplerob. I don't believe I've seen a button pattern used where we have different colored outline buttons side-by-side like this. Traditionally, we'd have a solid CTA style button next to an outline.
 
screen shot 2018-05-09 at 11 20 30 am

File: src/styles.scss [hunk 2]
```diff
@@ -122,4 +122,10 @@ app-neighbors {
   background-color: transparent;
   border-color: rgba(151, 151, 151, 0.75);
   color: #979797;
-}
\ No newline at end of file
+}
+
+.btn-cyan-light{
```
Comment (zarahzachz) [inline src/styles.scss]: @christiancrawford If you end up rolling your own button style here, be mindful that you'll need to support `:hover`, `:focus` and `:active` states. In the DDK, you can see the CSS behind our button code [here](https://github.com/crdschurch/crds-styles/blob/development/assets/stylesheets/_mixins.scss#L13)

File: src/app/components/pin-details/gathering/end-group/end-group-confirmation.component.html [hunk 3]
```diff
@@ -17,18 +17,14 @@
         </div>
 
         <footer class="row soft-top">
-          <div class="col-sm-6 col-sm-offset-3 btn-block-mobile text-center">
```
Comment (zarahzachz) [inline src/app/components/pin-details/gathering/end-group/end-group-confirmation.component.html]: @christiancrawford Why did you remove the `btn-block-mobile` class?

File: src/app/components/pin-details/gathering/end-group/end-group-confirmation.component.html [hunk 4]
```diff
@@ -17,18 +17,14 @@
...
-                    (click)="onEndGroup()">
+            <button type="button" class="btn btn-cyan" (click)="onEndGroup()">
               End group
             </button>
 
             <!-- Cancel Button: -->
-            <button type="button"
-                    class="btn btn-outline btn-default btn-block-mobile"
-                    (click)="onClose()">
+            <button type="button" class="btn btn-outline btn-default" (click)="onClose()">
```
Comment (zarahzachz) [inline src/app/components/pin-details/gathering/end-group/end-group-confirmation.component.html]: @christiancrawford Because of the button refactor in `crds-styles`, some of the classes may not map one-for-one here. To achieve the same button as we have in INT, it looks like the class `btn-gray-light` should be used instead of `btn-default`.

File: src/app/components/pin-details/gathering/gathering.html (hunk 1 above)
Comment (christiancrawford) [reply src/app/components/pin-details/gathering/gathering.html]: reverting this change per our conversation

File: src/styles.scss (hunk 2 above)
Comment (christiancrawford) [reply src/styles.scss]: reverting this change per our conversation

File: src/app/components/pin-details/gathering/end-group/end-group-confirmation.component.html (hunk 3 above)
Comment (christiancrawford) [reply src/app/components/pin-details/gathering/end-group/end-group-confirmation.component.html]: because I didn't have full understand on it, reverting this change

File: src/app/components/pin-details/gathering/end-group/end-group-confirmation.component.html (hunk 4 above)
Comment (christiancrawford) [reply src/app/components/pin-details/gathering/end-group/end-group-confirmation.component.html]: adjusted the styles here
